### ISSUE TITLE
Include bootstrap in Fee sub-menus

### ISSUE THREAD

**Issue by Trodrige** [LibreHealthIO/LibreEHR#840]
Add bootstrap to all elements under 'Fee' menu item, including sub-menus.

**Comment by MigDinny** [comment]
Mine

**Comment by MigDinny** [comment]
I give this task for @Jdew192837

**Comment by Jdew192837** [comment]
Thanks @Trodrige I will be furthering my bootstrap with these issues :D

**Comment by teryhill** [comment]
@jdew192837 are you still working on this?

**Comment by Jdew192837** [comment]
Yes I will be working on these soon, I will send them to you when I am done
so you can learn.

**Comment by Jdew192837** [comment]
@Trodrige Can you give some specifics about this, Thanks

**Comment by Trodrige** [comment]
@Jdew192837 look at the description again.
